package.json

```diff
@@ -34,12 +34,12 @@
   "dependencies": {
     "@hookform/resolvers": "^3.2.0",
     "@playwright/test": "^1.45.0",
-    "@tanstack/react-query": "4.29.7",
+    "@tanstack/react-query": "^4.29.7",
     "@tanstack/react-table": "^8.9.1",
     "axios": "^1.7.4",
     "byte-size": "^8.1.1",
     "classnames": "^2.3.2",
-    "design-system-react": "https://github.com/cfpb/design-system-react",
+    "design-system-react": "https://github.com/cfpb/design-system-react#918-mobile-menu",
```

Comment:
Heyo! Going to put this up on Dev again for Natalia, but going to put a hold on it just so we don't forget to remove this/merge it in on the DSR before we merge this PR.

Reply:
Mobile nav PR merged in the [DSR](https://github.com/cfpb/design-system-react/pull/380).  
 
This PR updated to point this dependency back at the `main` branch.